### PR TITLE
Adds OpenAI session

### PR DESCRIPTION
**Pull request by MattLuceen**
Adds the new-and-improved OpenAI session. The presentation was added with Git LFS as it is >200MB! Hopefully that doesn't break anything down the line (like displaying it in the pages site?) -- we'll see.

### CODE REVIEW

File: Activities/OpenAISession/README.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+## Agenda
```
Comment (josephmaxwellhyland) [inline Activities/OpenAISession/README.md:10]: Usually I will try to fill out the README a little more - you don't have to add too much, but just think of what another person would need to know if they were leading this session. Could be as simple as [this](https://github.com/hylandtechoutreach/hackathon/tree/main/Activities/UxSession), plus your agenda.

**Review by josephmaxwellhyland — COMMENTED** [review]
Overall everything looks great! Just need a little more information in the README :)

File: Activities/OpenAISession/README.md (hunk 1 above)
Comment (MattLuceen) [reply Activities/OpenAISession/README.md:10]: Thanks! Done!

**Review by josephmaxwellhyland — APPROVED** [review]
looks great to me!

**Pull request merged**